Hunting Queries/SecurityEvent/RareProcbyServiceAccount.txt

```diff
@@ -0,0 +1,61 @@
+// Name: Rare processes run by Service accounts
+// Description: Service accounts normally are supposed to perform a limited set of tasks in a stable environment. The query collects a list of service account and then joins them with rare processes in an environment to detect anomalous behaviours.
```

Comment:
May want to consider making this 2 lines - 
// Description: Service accounts normally are supposed to perform a limited set of tasks in a stable environment. 
// The query collects a list of service account and then joins them with rare processes in an environment to detect anomalous behaviours.

Reply:
Thanks. Made the changes.